src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.component.html

```diff
@@ -112,18 +117,33 @@
                                 </mat-radio-button>
                             </mat-radio-group>
                         </div>
-                        <div class="select-item d-flex">
+                        <div *ngIf="!accountView" class="select-item d-flex position-relative">
```

Comment:
Please explain why we have this `ngIf` here. It seems searching by account exists in both modes.

Reply:
Search by account should not be in the account

Reply:
No, as we've just found out recently.